File: Code/Cplus/LAB12_07/LAB12_07.cpp
#include <iostream>
#include <fstream>
#include <string>
using namespace std;

int main() {
    double a, b;
    cout << "Enter first number: ";
    cin >> a;
    cout << "Enter second number: ";
    cin >> b;

    ofstream outFile("calculation.txt");
    if (outFile.is_open()) {
        outFile << a << " + " << b << " = " << (a + b) << endl;
        outFile << a << " - " << b << " = " << (a - b) << endl;
        outFile << a << " * " << b << " = " << (a * b) << endl;
        outFile << a << " / " << b << " = " << (b != 0 ? a / b : 0) << endl;
        outFile.close();
        cout << "[บันทึกผลลัพธ์เรียบร้อย!]\n";
    }

    ifstream inFile("calculation.txt");
    if (inFile.is_open()) {
        string line;
        cout << "\n[อ่านข้อมูลจากไฟล์ calculation.txt]\n";
        while (getline(inFile, line)) {
            cout << line << endl;
        }
        inFile.close();
    }

    return 0;
}

File: Code/Python/LAB12_07.py
# รับค่าตัวเลขจากผู้ใช้
a = float(input("Enter first number: "))
b = float(input("Enter second number: "))

# คำนวณผลลัพธ์
calculations = [
    f"{a} + {b} = {a + b}",
    f"{a} - {b} = {a - b}",
    f"{a} * {b} = {a * b}",
    f"{a} / {b} = {a / b if b != 0 else 'undefined'}"
]

# บันทึกผลลัพธ์ลงไฟล์
with open("calculation.txt", "w") as file:
    file.write("\n".join(calculations))
print("[บันทึกผลลัพธ์เรียบร้อย!]")

# อ่านข้อมูลจากไฟล์
print("\n[อ่านข้อมูลจากไฟล์ calculation.txt]")
with open("calculation.txt", "r") as file:
    print(file.read())
